.github/workflows/esti.yaml

```diff
@@ -1211,6 +1220,13 @@ jobs:
           ESTI_STORAGE_NAMESPACE: https://esti4hns.blob.core.windows.net/esti-system-testing/${{ github.run_number }}/${{ steps.unique.outputs.value }}
           ESTI_ADLS_IMPORT_BASE_URL: https://esti4hns.adls.core.windows.net/esti-system-testing-data/
 
+      - name: cleanup cosmos db container
+        if: always()
+        continue-on-error: true
+        run: |
+          echo "Delete database container: ${{ steps.unique.outputs.cosmos_db_container }}"
+          az cosmosdb sql container delete -a ${{ env.COSMOSDB_ACCOUNT }} -g ${{ env.COSMOSDB_ACCOUNT }} -d ${{ env.COSMOSDB_DATABASE }} -n ${{ steps.unique.outputs.cosmos_db_container }} --yes
+
```

Comment:
One obvious worry here is that this deletes the evidence when the test fails.  Usually this doesn't matter, but sometimes it is _very_ helpful.

Possible mitigations:
* Dump DB contents (always or at least on failure) before deleting it.
* Find some way to inject an option _not_ to delete.  For instance this might be an additional flag that I can set when I trigger Esti manually (oh yeah, you'll also be adding that...)

Of these two I prefer the first, especially if it is possible to reload the dump.  But I am happy for other options of course!

Feel free to defer this to a tech debt issue

Reply:
We currently don't do this for the Postgres and DynamoDB (we use a docker image for both).
I think this could be a good idea - but we'll have to dump the db somewhere else and not leave it otherwise we will find ourselves in the same problem.
Anyway opened https://github.com/treeverse/lakeFS/issues/6112